src/selectors/layout.test.js

```diff
@@ -1,10 +1,37 @@
-import { mockState } from '../utils/state.mock';
-import { getChartSize, getSidebarWidth, getGraphInput } from './layout';
+import { mockState, prepareState } from '../utils/state.mock';
+import {
+  getChartSize,
+  getSidebarWidth,
+  getGraphInput,
+  getTriggerLargeGraphWarning,
+} from './layout';
 import { updateFontLoaded } from '../actions';
 import reducer from '../reducers';
-import { sidebarWidth } from '../config';
+import { sidebarWidth, largeGraphThreshold } from '../config';
+import animals from '../utils/data/animals.mock.json';
 
 describe('Selectors', () => {
+  describe('getTriggerLargeGraphWarning', () => {
```

Comment:
(optional) It'd be great to add tests to check that this will show when `!ignoreLargeWarning` or `!graphHasNodes`, but no big deal, we can add this later :)